packages/client/src/pages/create-card/CreateCard.tsx

```diff
@@ -1,7 +1,27 @@
+import { z } from 'zod';
+
+import Form from '../../components/form/Form';
+import FormButton from '../../components/form/form-button/FormButton';
+import { useForm } from '../../components/form/hook/useForm';
+import Input from '../../components/form/input/Input';
+
+const createCardSchema = z.object({
+  firstName: z.string().min(1, 'First Name must be at least 1 characters long!'),
+  lastName: z.string().min(1, 'Last Name must be at least 1 characters long!'),
+});
+
 export const CreateCard = () => {
+  const form = useForm({
+    schema: createCardSchema,
+  });
+
   return (
     <>
-      <h1>This is a Create Card page</h1>
+      <Form form={form} onSubmit={(values) => console.log(values)}>
+        <Input label="First Name" {...form.register('firstName')} />
+        <Input label="Last Name" {...form.register('lastName')} />
+        <FormButton name="Create Card" />
```

Comment:
Unnecessary `name` prop, I reader wishes to have a more flexible way to forward maybe something different regarding string. E.g. icon, loader, some small cmp.
So use children instead :)

Reply:
I will try to reuse the existing Button cmp with children props, in that case, this issue would be resolved.